examples/adjoints/adjoints_ex7/adjoint_initial.C

```diff
@@ -0,0 +1,77 @@
+// The libMesh Finite Element Library.
+// Copyright (C) 2002-2020 Benjamin S. Kirk, John W. Peterson, Roy H. Stogner
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+
+
+#include "adjoint_initial.h"
+
+using namespace libMesh;
+
+void adjoint_read_initial_parameters()
+{
+}
+
+void adjoint_finish_initialization()
+{
+}
+
+
+
+// Initial conditions
+Number adjoint_initial_value0(const Point & p,
+                             const Parameters &,
+                             const std::string &,
+                             const std::string &)
+{
+  Real x = p(0), y = p(1);
```

Comment:
Okay, I'm going to stop here for now, at the unused variables.  I need to push a re-bootstrap commit so that our CI actually runs this example, and you need to reconfigure with --enable-paranoid-warnings --enable-werror so that the compiler can have more to harass you about and I can have less.

Reply:
Yeah, some cleanup is needed.